utils/db/setup.ts

```diff
@@ -1,14 +1,13 @@
 import conn from '.';
 
-export const getSetupReposFromDbForUserId = async (userId: string, org: string, provider: string): Promise<string[]> => {
-	console.log(`[getSetupReposFromDbForUserId] Getting setup repos from db for ${userId} and org ${org} and provider ${provider}`);
+export const getSetupReposFromDbForOrg = async (org: string, provider: string): Promise<string[]> => {
```

Comment:
```suggestion
export const getSetupReposFromDbForOwner = async (owner: string, provider: string): Promise<string[]> => {
```

Using the word "org" is misguiding because this can also me an individual owner.